src/main/java/core/basesyntax/service/impl/CsvFileParserImpl.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileParser;
+
+public class CsvFileParserImpl implements FileParser {
+    @Override
+    public String[] parseData(String line) {
+        return line.split(",");
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
```suggestion
        return line.split(SEPARATOR);
```